src/main/components/form/validator.ts

```diff
@@ -335,20 +180,10 @@ export const isValidCurrency: Validator = value => {
   }
 };
 
-let otherGenderTitle: string | string[];
-
-export const validateGenderTitle: Validator = value => {
-  otherGenderTitle = value;
-};
-
-export const validatePreferredOther: Validator = value => {
-  let outcome: string = undefined;
-  if (otherGenderTitle === 'Other') {
-    if ((value as string).trim().length === 0) {
-      outcome = 'required';
-    } else if (/^\d+$/.test(value as string) || /^\s*\d/.test(value as string)) {
-      outcome = 'numberError';
-    }
-    return outcome;
+export const validateTitlePreference: Validator = value => {
```

Comment:
Would it work to declare value as a string in the signature? (ie ` = (value: string) => {`). Would eliminate needing to "cast" it multiple times in the body

Reply:
That would be much a larger change and not as part of this commit, I've only just added another Validator with its existing signature.